SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -76,14 +76,14 @@ In this section you’ll run a [local deployment](https://docs.improbable.io/ref
 
 1. In a text editor, open `UnrealGDKThirdPersonShooter\spatial\default_launch.json` and in the `layer_configurations` section set `rows` to `2`. This instructs SpatialOS that you will be launching two server-workers.
```

Comment:
The GDK default is to use the generated launch config, so changing this variable should be done via the GDK settings. I'd suggest removing this line, and I'll add one after setting the number of servers.